uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/PaymentMVP.kt

```diff
@@ -35,12 +37,14 @@ interface PaymentMVP {
 
         fun initialiseGuestPayment(price: QuotePrice?)
 
-        fun sdkInit()
+        fun sdkInit(price: QuotePrice?)
 
         fun passBackNonce(braintreeSDKNonce: String)
 
         fun updateCardDetails(nonce: String, description: String, typeLabel: String)
 
+//        fun handleAddCard()
```

Comment:
Does this need to be left commented out? Something to change/be added later on I assume?

Reply:
Deleted